src/components/FieldImage.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getTranslator } from '../reducers/translation';
+import styles from './FieldImage.scss';
+
+const fileInstance = new FileReader(); // eslint-disable-line no-undef
+const maxFilesize = 3000000;
+
+const DropzoneImage = ({ i18n, input }) => (
+  <div>
+    <Dropzone
+      accept={'image/*,application/pdf'}
+      maxSize={maxFilesize}
+      multiple={false}
+      className={styles.overide}
+      name={input.name}
+      onDropAccepted={(imageData) => {
+        fileInstance.readAsDataURL(imageData[0]);
```

Comment:
The same `fileInstance` is reused, this can cause a race condition.

Solution: A new instance needs to be created for every call to `onDropAccepted`.